iroha-balancer-integration-test/src/test/kotlin/soramitsu/iroha/balancer/integration/IntegrationTests.kt

```diff
@@ -0,0 +1,121 @@
+import java.util.concurrent.TimeUnit
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class IntegrationTests {
+
+    private val containerHelper = ContainerHelper()
+
+    private val irohaBalancerCoreContextFolder = "${containerHelper.userDir}/../iroha-balancer-core/build/docker/"
+    private val irohaBalancerCoreDockerfile = "${containerHelper.userDir}/../iroha-balancer-core/build/docker/Dockerfile"
+
+    private val irohaBalancerCoreContainer = containerHelper
+            .createIrohaBalancerCoreContainer(
+                    irohaBalancerCoreContextFolder,
+                    irohaBalancerCoreDockerfile
+            )
+
+    private val subscriptionStrategy = WaitForTerminalStatus(
+            listOf(
+                    TxStatus.COMMITTED,
+                    TxStatus.REJECTED,
+                    TxStatus.UNRECOGNIZED
+            )
+    )
+    private val crypto = Ed25519Sha3()
+    private lateinit var client: IrohaBalancerClientService
+
+    @BeforeAll
+    fun init() {
+        TestContainersMock.start()
+
+        irohaBalancerCoreContainer
+                .withEnv("CAMEL_COMPONENT_RABBITMQ_HOSTNAME", rmqConfig.host)
+                .withEnv("CAMEL_COMPONENT_RABBITMQ_PORT_NUMBER", rmqConfig.port.toString())
+                .withEnv("CAMEL_COMPONENT_RABBITMQ_USERNAME", "guest")
+                .withEnv("CAMEL_COMPONENT_RABBITMQ_PASSWORD", "guest")
+                .withEnv("IROHA_PEERS", TestContainersMock.getPeerAddresses())
+                .waitingFor(Wait.forLogMessage(".*Started IrohaBalancerApplication.*", 1))
+                .start()
+
+        client = IrohaBalancerClientService(rmqConfig)
+    }
+
+    @AfterAll
+    fun tearDown() {
+        client.close()
+        irohaBalancerCoreContainer.close()
+        TestContainersMock.stop()
+    }
+
+    @Test
+    fun createAccountTransactionTest() {
+        val transaction: TransactionOuterClass.Transaction = Transaction.builder(defaultAccountId)
+                .createAccount("account_", defaultDomainName, crypto.generateKeypair().public)
+                .build()
+                .sign(defaultKeyPair)
+                .build()
+        client.balanceToTorii(transaction)
+
+        val iroha = IrohaAPI(network.toriiAddresses[0])
+        checkCommitted(transaction, iroha)
+    }
+
+    @Test
+    fun batchTransactionTest() {
+        val transaction1: TransactionOuterClass.Transaction = Transaction.builder(defaultAccountId)
+                .createAsset("usd", defaultDomainName, 2)
+                .build()
+                .sign(defaultKeyPair)
+                .build()
+
+        val transaction2: TransactionOuterClass.Transaction = Transaction.builder(defaultAccountId)
+                .createAsset("khr", defaultDomainName, 0)
+                .build()
+                .sign(defaultKeyPair)
+                .build()
+        val transactions = listOf(transaction1, transaction2)
+
+        client.balanceToListTorii(transactions)
```

Comment:
it's not batch actually, but just a list of transactions 😄  please rewrite that to a batch